### ISSUE TITLE
WCAG 2.1 compliancy: Hide tooltip on ESC

### ISSUE THREAD

**Issue by StudioVDS** [vimeo/player.js#804]
### Expected Behavior
When focussing using TAB inside the embedded player to one of the buttons (Like, Add to watch layer, Share) on the right side of the player. Tooltips appear. You should be able to hide these tooltips by using the ESC button [WCAG 1.4.13](https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html).

**Comment by rkrishnan8594** [comment]
Hi @StudioVDS - thanks for reporting this issue. It's been ticketed internally for later prioritization.
